packages/xstate-react/README.md

```diff
@@ -126,7 +132,7 @@ A [React hook](https://reactjs.org/hooks) that subscribes to state changes from
 - `state` - Represents the current state of the service as an XState `State` object.
 - `send` - A function that sends events to the running service.
 
-### `useActor(actor, getSnapshot)`
+### `useActor(actor, getSnapshot?)`
```

Comment:
q: should we support `getSnapshot` in the future? or should we just expect that `actor.getSnapshot` is available?

Reply:
We should not (but this is a major breaking change). We should expect `.getSnapshot` to be available.